npm/react/package.json

```diff
@@ -16,26 +16,21 @@
     "watch": "yarn build --watch"
   },
   "dependencies": {
-    "@babel/plugin-transform-modules-commonjs": "7.12.1",
-    "@cypress/code-coverage": "3.8.6",
     "@cypress/webpack-preprocessor": "0.0.0-development",
-    "babel-plugin-istanbul": "6.0.0",
     "debug": "4.3.2",
-    "find-up": "5.0.0",
-    "find-webpack": "2.2.1",
-    "mime-types": "2.1.26",
-    "semver": "7.3.2",
-    "unfetch": "4.1.0"
+    "find-webpack": "2.2.1"
```

Comment:
I think we should prob move away from `find-webpack` entirely eventually - it's only used in 4 places atm.

Reply:
@lmiller1990 we need some api for users to not look for webpack config manually for different options – create-react-app/next/nuxt. Once we will implement something fro this – we will be able to remove it :)